api/src/test/java/com/doordeck/sdk/util/JWTSplitTest.java

```diff
@@ -0,0 +1,24 @@
+package com.doordeck.sdk.util;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Splitter;
+
+import org.junit.Test;
+
+import static com.google.common.base.Strings.lenientFormat;
+
+public class JWTSplitTest {
+
+    private String jwt = "";
+
+    @Test
+    public void splitJWT() {
+        try{
+            String val = jwt.substring(jwt.indexOf(".")+1, jwt.lastIndexOf("."));
+            System.out.println(val);
+        } catch (StringIndexOutOfBoundsException e) {
+            throw new IllegalArgumentException("Incorrect JWT TOKEN");
+        }
+    }
```

Comment:
What exactly is the test trying to do?  Since the JWT input is an empty string it'll always throw an IllegalArgumentException which will mean the tests fails.

If you're testing the exception, you should add @Test(expected = IllegalArgumentException.class)

Reply:
I was testing with different incorrect JWT strings

Reply:
Do you just wanna delete this then?